.github/workflows/benchmark-pipeline.yaml

```diff
@@ -80,12 +80,12 @@ jobs:
         run: |
           kubectl apply -f manifest.yaml
 
-          sleep 10
+          sleep 20
 
           kubectl wait pod \
           --all \
           --for=condition=Ready \
-          --namespace=falco # TODO: Revert to "benchmark" this after merging https://github.com/falcosecurity/cncf-green-review-testing/pull/22
+          --namespace=benchmark
```

Comment:
Wait should match target namespace 
https://github.com/cncf-tags/green-reviews-tooling/blob/ae76ef9b13bb4e159e0ae56bae1f5f83f978e0a0/projects/falco/ebpf.yaml#L25

Reply:
I see - it should be ok if we have falco deployed in namespace `benchmark` and the benchmark tests in namespace `falco`, temporarily. A bit confusing but we should be able to revert it soon. 🤞

Reply:
Yes it is a bit confusing but once your PR in the falco repo is merged we can tidy up and just use the benchmark namespace.